### PR TITLE
fix: (llvm-git, lib32-llvm-git) the reasons why they can't install 

### PR DESCRIPTION
**Pull request by SolarAquarion**
I'm also adding more files in clang-git/usr/bin fixing conflicts in pacman

Next, i'm going to push mesa fixes/updates

### CODE REVIEW

File: archlinuxcn/lib32-llvm-git/PKGBUILD [hunk 1]
```diff
@@ -4,7 +4,7 @@
 pkgbase=lib32-llvm-git
 pkgname=('lib32-llvm-libs-git' 'lib32-clang-git' 'lib32-llvm-git')
 pkgdesc='Low Level Virtual Machine (git version)'
-pkgver=16.0.0_r432160.786b503f66b1a
+pkgver=15.0.0_r425063.0bd645d3
```
Comment (SilverRainZ) [inline archlinuxcn/lib32-llvm-git/PKGBUILD:7]: This pkgver is older than the previous one, is this expected? Or you should bump the epoch.

**Comment by SolarAquarion** [comment]
No, that was a mistake.  The runner should update the pkgver

**Comment by SolarAquarion** [comment]
@SilverRainZ lilac run should bump the pkgver

File: archlinuxcn/mesa-git/PKGBUILD [hunk 2]
```diff
@@ -91,7 +91,7 @@ package_vulkan-mesa-layers-git() {
   provides=('vulkan-mesa-layers')
   depends=('wayland' 'libxcb' 'python')
   conflicts=('vulkan-mesa-layer-git' 'vulkan-mesa-layers')
-  replaces=('vulkan-mesa-layer-git')
+  replaces=('vulkan-mesa-layer')
```
Comment (yuyichao) [inline archlinuxcn/mesa-git/PKGBUILD:94]: You must not replace official package and VCS package must also not replace non-VCS packages in general.

**Review by yuyichao — CHANGES_REQUESTED** [review]

File: archlinuxcn/mesa-git/PKGBUILD [hunk 3]
```diff
@@ -91,7 +91,6 @@ package_vulkan-mesa-layers-git() {
   provides=('vulkan-mesa-layers')
   depends=('wayland' 'libxcb' 'python')
   conflicts=('vulkan-mesa-layer-git' 'vulkan-mesa-layers')
-  replaces=('vulkan-mesa-layer-git')
```
Comment (yuyichao) [inline archlinuxcn/mesa-git/PKGBUILD:94]: Assuming he package used to be called vulkan-mesa-layer (without s) this should stay as is.

File: archlinuxcn/lib32-llvm-git/PKGBUILD (hunk 1 above)
Comment (yuyichao) [reply archlinuxcn/lib32-llvm-git/PKGBUILD:7]: There’s no other change to this file? Then this is only going to cause conflicts….

File: archlinuxcn/llvm-git/PKGBUILD [hunk 4]
```diff
@@ -173,7 +173,7 @@ package_compiler-rt-git() {
 
     local _llvmver=$(echo "$pkgver" | grep -Po "(^[\.\d]+)")
 
-    _fakeinstall fakeinstall/usr/lib/clang/"$_llvmver"/lib/linux
```
Comment (yuyichao) [inline archlinuxcn/llvm-git/PKGBUILD:176]: Why the change of path? The none git version seems to be installing to linux like the old version rather than CHOST.

**Review by yuyichao — CHANGES_REQUESTED** [review]

File: archlinuxcn/llvm-git/PKGBUILD (hunk 4 above)
Comment (SolarAquarion) [reply archlinuxcn/llvm-git/PKGBUILD:176]: it on my system and other build hosts was saying that there was no such thing as blah blah blah, and i realized that it changed it's logic to $CHOST

**Review by yuyichao — APPROVED** [review]

**Comment by yuyichao** [comment]
Please avoid changing the pkgvers in PR since they can easily create merge conflicts.

Also, please don't include merge commits in PR since they are more difficult to be rebased out especially when they are there due to merge conflicts. I've rebased manually.
